libs/util/shared/src/lib/services/theme.service.ts

```diff
@@ -89,6 +91,13 @@ export class ThemeService {
     applyColor('main-800', scale(0.8))
     applyColor('main-900', scale(0.9))
 
+    if (titleColor) {
+      applyColor('title', chroma(titleColor))
+    }
+    if (titleHeaderColor) {
+      applyColor('title-header', chroma(titleHeaderColor))
```

Comment:
There is no need to define a CSS variable for this setting; the header text color can just be read directly from the settings in the header component, thus we end up with a much small scale change

Reply:
Ok, I'll have a look at this.